### ISSUE TITLE
The label of the dataset is not binary map.

### ISSUE THREAD

**Issue by yiyi-today** [liumency/DSAMNet#5]
Thank you for your generous sharing！

I downloaded the CDD dataset mentioned in your paper, but found that the label of the CDD dataset was not binary map, as Issue in label dataset of 2018. Lebedev M A, Vizilter Y V, Vygolov O V, et al. Change detection in remote sensing images using conditional adversarial networks said.

How did you deal with this situation ?

thanks

**Comment by WesternTrail** [comment]
Yes, I also encountered this problem. During debugging, I found that there are three types of label values: - 1, 0, and 1. This caused me to make an error when calculating the precision？Have you solved the problem？？

**Comment by WesternTrail** [comment]
@yiyi-today @liumency

**Comment by WesternTrail** [comment]
In fact, there is no problem with the dataset. I speculate that you encountered this problem when calculating precision. After debugging for a long time, I found that the problem was that when BCL() was called, the label was passed in, but the label was changed in the BCL() function. Therefore, when calculating the precision, if you directly compare the pred with the label, there will be a problem here. So the correct approach is to use label. clone () for deep copying in advance!
@yiyi-today
